src/stores/guestName.ts

```diff
@@ -0,0 +1,118 @@
+/**
+ * SPDX-FileCopyrightText: 2019 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+import { getGuestNickname, setGuestNickname } from '@nextcloud/auth'
+import { t } from '@nextcloud/l10n'
+import { defineStore } from 'pinia'
+import { ref } from 'vue'
+import { setGuestUserName } from '../services/participantsService.js'
+import { useActorStore } from './actor.ts'
+
+type AddGuestNamePayload = { token: string, actorId: string, actorDisplayName: string }
+
+const LOCALIZED_GUEST = t('spreed', 'Guest')
+
+export const useGuestNameStore = defineStore('guestName', () => {
```

Comment:
Micro optimization - lazy translation
```suggestion
export const useGuestNameStore = defineStore('guestName', () => {
	const LOCALIZED_GUEST = t('spreed', 'Guest')
```